enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchLongPatternAcceptanceTest.scala

```diff
@@ -61,7 +61,8 @@ class MatchLongPatternAcceptanceTest extends ExecutionEngineFunSuite with QueryS
 
     // WHEN
     val idpInnerIterations = determineIDPLoopSizes(numberOfPatternRelationships,
-      cypher_idp_solver_table_threshold, maxTableSizes)
+      cypher_idp_solver_table_threshold, maxTableSizes, Map(cypher_idp_solver_duration_threshold -> "10000"))
```

Comment:
as other comment below.
default for `cypher_idp_solver_duration_threshold` is 1000. why do we run with non-default value?
if anything, wouldn't increasing it by 10x increase the duration of planning?